working-groups/WORKING-GROUPS.md

```diff
@@ -55,7 +55,7 @@ The current working groups are:
 
 - [API Core](#api-core)
 - [Client](#client)
-- [Documentation](#documentation)
+- [Documentation + User Experience](#documentation-+-user-experience)
```

Comment:
```suggestion
- [Documentation + User Experience](#documentation--user-experience)
```

I think this is the proper link syntax (just tried it)

Reply:
I was wondering about this! Thank you